packages/lending/test/transactions/zap-repay.test.ts

```diff
@@ -27,6 +27,7 @@ describe('Transaction: Zap Repay', function () {
     await claimToken(chainId, '0xaf0FDd39e5D92499B0eD9F68693DA99C0ec1e92e', mainnetTokens.USDT, '2000');
     await claimToken(chainId, '0x06e4Cb4f3ba9A2916B6384aCbdeAa74dAAF91550', mainnetTokens.USDT, '2000');
     await claimToken(chainId, '0x53fb0162bC8d5EEc2fB1532923C4f8997BAce111', mainnetTokens.USDT, '2000');
+    await claimToken(chainId, '0x8bf7058bfe4cf0d1fdfd41f43816c5555c17431d', mainnetTokens.USDC, '2000');
```

Comment:
Just think if we can keep USDT(wallet) to USDC(debt) same as other cases, it would looks neat in whole file.